pkg/services/accesscontrol/evaluator/evaluator_test.go

```diff
@@ -45,49 +50,54 @@ func TestEvaluatePermissions(t *testing.T) {
 		{
 			Name: "No expected scope always returns true",
 			HasScopes: map[string]struct{}{
-				"teams:*/permissions:*": {},
-				"users:*":               {},
-				"permissions:delegate":  {},
+				"teams:*":              {},
+				"permissions:*":        {},
+				"users:*":              {},
+				"permissions:delegate": {},
 			},
 			NeedAnyScope: []string{},
 			Valid:        true,
 		},
 		{
 			Name: "Single scope from  list",
 			HasScopes: map[string]struct{}{
-				"teams:1/permissions:delegate": {},
+				"teams:1":              {},
+				"permissions:delegate": {},
 			},
-			NeedAnyScope: []string{"teams:1/permissions:delegate"},
+			NeedAnyScope: []string{"teams:1", "permissions:delegate"},
```

Comment:
This kind of depends on #35996 and doesn't need an immediate fix, but we'd want this to be "need all scopes".